doc/Command_Index_gregorio.tex

```diff
@@ -181,58 +181,58 @@ \section{Gregorio Controls}
 Macro to typeset a divisio finalis.
 
 \begin{argtable}
-  \#1 & code & Macros which may happen before the skip but after the divisio finalis (typically \verb=\grevepisemus=).\\
+  \#1 & \TeX\ code & Macros which may happen before the skip but after the divisio finalis (typically \verb=\grevepisemus=).\\
 \end{argtable}
 
 \macroname{\textbackslash GreDivisioMaior}{\#1}{gregoriotex-signs.tex}
 Macro to typeset a divisio maior.
 
 \begin{argtable}
-  \#1 & code & Macros which may happen before the skip but after the divisio maior (typically \verb=\grevepisemus=).\\
+  \#1 & \TeX\ code & Macros which may happen before the skip but after the divisio maior (typically \verb=\grevepisemus=).\\
 \end{argtable}
 
 \macroname{\textbackslash GreDivisioMinima}{\#1}{gregoriotex-signs.tex}
 Macro to typeset a divisio minima.
 
 \begin{argtable}
-  \#1 & code & Macros which may happen before the skip but after the divisio minima (typically \verb=\grevepisemus=).\\
+  \#1 & \TeX\ code & Macros which may happen before the skip but after the divisio minima (typically \verb=\grevepisemus=).\\
 \end{argtable}
 
 \macroname{\textbackslash GreDivisioMinor}{\#1}{gregoriotex-signs.tex}
 Macro to typeset a divisio minor.
 
 \begin{argtable}
-  \#1 & code & Macros which may happen before the skip but after the divisio minor (typically \verb=\grevepisemus=).\\
+  \#1 & \TeX code & Macros which may happen before the skip but after the divisio minor (typically \verb=\grevepisemus=).\\
 \end{argtable}
 
 \macroname{\textbackslash GreDominica}{\#1\#2}{gregoriotex-signs.tex}
 Macro to typeset a dominican bar.
 
 \begin{argtable}
-  \#1 & integer 1-- 6& Type of dominican bar.  Corresponds to bar types 6--11 in \verb=\grewritebar=.\\
-  \#2 & code    & Macros which may happen before the skip but after the divisio dominica (typically \verb=\grevepisemus=).\\
+  \#1 & \texttt{1--6} & Type of dominican bar.  Corresponds to bar types 6--11 in \verb=\grewritebar=.\\
```

Comment:
Should the dash be in teletype font?  Why dash here and not slashes as in the other file?